change/office-ui-fabric-react-2019-11-25-16-27-13-teachingbubble-fix.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "TeachingBubble: margin for header no longer is hidden under a close button.",
```

Probably update this too
```suggestion
  "comment": "TeachingBubble: add hasCloseButton and deprecate hasCloseIcon; prevent header text from overlapping with close button.",
```